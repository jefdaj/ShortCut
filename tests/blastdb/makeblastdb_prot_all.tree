.
|-- cache
|   |-- load
|   |   |-- 8a695c317a.faa
|   |   |-- 8a695c317a.faa.ids
|   |   |-- 8aa4a9d2fc.faa
|   |   `-- 8aa4a9d2fc.faa.ids
|   `-- makeblastdb
|       `-- f01020ac88
|           |-- f01020ac88.pdb.phr
|           |-- f01020ac88.pdb.pin
|           `-- f01020ac88.pdb.psq
|-- exprs
|   |-- list
|   |   `-- 40bb17a71b.faa.list -> ../../cache/lines/...
|   |-- load_faa
|   |   |-- 498816b813.faa -> ../../cache/load/8aa4a9d2fc.faa
|   |   `-- 70717c8ea5.faa -> ../../cache/load/8a695c317a.faa
|   |-- makeblastdb_prot_all
|   |   `-- c51c1a6374.pdb -> ../../cache/lines/...
|   `-- str
|       |-- 50dbefb6d6.str -> ../../cache/lines/...
|       `-- ec3b181d8a.str -> ../../cache/lines/...
`-- vars
    |-- db.pdb -> ../exprs/makeblastdb_prot_all/c51c1a6374.pdb
    |-- large6803faa.faa -> ../exprs/load_faa/498816b813.faa
    |-- result -> ../vars/db.pdb
    `-- small6803faa.faa -> ../exprs/load_faa/70717c8ea5.faa

10 directories, 17 files
