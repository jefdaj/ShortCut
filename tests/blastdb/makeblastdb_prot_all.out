Database: f01020ac88.pdb
3,574 sequences; 1,140,592 total residues
Longest sequence: 4,199 residues
Volumes:
$TMPDIR/cache/makeblastdb/f01020ac88/f01020ac88.pdb

